style.css

```diff
@@ -505,6 +588,27 @@ footer {
 
 /* Medium devices (landscape tablets, 768px and up) */
 @media only screen and (min-width: 768px) {
+  #popup {
+    padding-top: 4rem;
+  }
+
+  #popup-close {
+    background-color: var(--glass);
+    padding: 0.5rem;
+    border-radius: 0.5rem;
+    top: 1rem;
+    right: 1rem;
+  }
+
+  .popup-title {
+    grid-area: 2 / span 1;
+  }
+
+  #popup-buttons-container {
+    grid-area: 2;
+    justify-content: flex-end;
+  }
```

Comment:
- [ ] Hi @BahirHakimy, Great effort working on the popup modal section of your portfolio project 👍, But I would like to raise some of the issues that needed your attention. Below are some of the highlights of the things that you need to work on.

 - I noticed when the `see project` is clicked on to display the popup card, the card appears at the top of the page and when scrolling down the main page of the project is shown underneath which is not supposed to be like that. the popup card should pop up immediately after the `see project` is clicked on and be fitted to the center of the screen as seen in the Figma design.
 - I would kindly advise you to give the `backdrop-filter: blur(10px);` CSS property to the popup container in order to give the main page a kind of a blur appearance when the popup card is displayed as seen in the Figma design.
 - On the `mobile view`, the `see live` and `see source` are not properly aligned with their respective `icons` as seen in the Figma design.
 - I would kindly suggest you stick to the Figma design as much as possible in such a way that once the popup card is displayed, the main page of the project can't access again (i.e should not show again).


Your Design 👇 

| Desktop  | mobile |
| ------------- | ------------- |
| ![bes-des](https://user-images.githubusercontent.com/78634841/216460539-22a4607e-1afb-4feb-918e-f45f0f1374c0.png)  | ![bes-mob](https://user-images.githubusercontent.com/78634841/216460600-b6d31cbf-5d1a-4ed9-a07f-afa5d0d9008c.png) |
 

Figma Design 👇 

| Desktop  | mobile |
| ------------- | ------------- |
| ![pp3-real](https://user-images.githubusercontent.com/78634841/216460904-1c1468df-2de5-4b39-bf3b-d06f0f272bb3.png) | ![pp3-mob](https://user-images.githubusercontent.com/78634841/216461244-83c99a24-0b9a-4348-ad53-b4d3cd4c351e.png) |